website_medical/static/src/js/file_upload_text_insert.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2016 LasLabs Inc.
+// License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+$(document).ready(function() {
```

Comment:
Oh interesting. When I was working on the button earlier in the ticket I saw that more as aesthetics rather than core logic for uploading.

Fortunately didn't really take much time to do and will keep this in mind for next time. Do you want me to remove the file?

Also instead of `$(document).ready`, should that be `base.ready().then(function(){` instead? Looks like I have some reviewing to do in terms of Odoo JS.